code/game/objects/items.dm

```diff
@@ -116,6 +124,16 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 	if(GLOB.rpg_loot_items)
 		rpg_loot = new(src)
 
+	LAZYINITLIST(default_strings)
+	//populate default_strings
+	default_strings += "very low"
+	default_strings += "low"
+	default_strings += "medium"
+	default_strings += "<font color=green>high</font>"

```

Comment:
you shouldn't actually do colors, since you can't make sure they're visible on every possible tooltip style